scss/components/_activity-no-result-icon.scss

```diff
@@ -0,0 +1,15 @@
+.civicase__no-result-icon {
```

Comment:
These icons are only for activity right?
The component name does not denote that.
Should be 
`civicase__activity-no-result-icon` - this one should have the SVG for Activities which is not milestone.
`civicase__activity-no-result-icon--milestone`
`civicase__activity-no-result-icon--task` (no need to create now, but for future)
`civicase__activity-no-result-icon--communication` (no need to create now, but for future)

Lastly, this is not a new component, please place them under `ActivityCard.scss`

Reply:
> Should be `civicase__activity-no-result-icon` - Noted 
>  this is not a new component, please place them under `ActivityCard.scss` - Noted

> this one should have the SVG for Activities which is not milestone. 
I think according to BEM. There should be a base class which contains all the basic generic styles
and then these styles should have different variants.
so it should be something like this
```
.civicase__activity-no-result-icon {
  background-position: center center;
  background-repeat: no-repeat;
  background-size: contain;
  height: 48px;
  width: 48px;
}

.civicase__activity-no-result-icon--milestone {
  background-image: url('../resources/icons/milestone.svg');
}

.civicase__activity-no-result-icon--activity {
  background-image: url('../resources/icons/activities.svg');
}

.civicase__activity-no-result-icon--activity {
  background-image: url('../resources/icons/activities.svg');
}

.civicase__activity-no-result-icon--activity {
  background-image: url('../resources/icons/activities.svg');
}

```